src/pages/ReportDetailsPage.tsx

```diff
@@ -160,7 +160,7 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
                 isAnonymousAction: false,
                 shouldShowRightIcon: true,
                 action: () => {
-                    if (isUserCreatedPolicyRoom || isChatThread || isPolicyExpenseChat) {
+                    if (isUserCreatedPolicyRoom || isChatThread || (isPolicyExpenseChat && isPolicyAdmin)) {
```

Comment:
Why are we preventing people other than policy admins from accessing the members page?

Reply:
I thought this change only involved preventing the removal of default members from workspace chats.

Reply:
> Why are we preventing people other than policy admins from accessing the members page?

_"Invited members(invited by members) are not able to invite or remove from the workspace chat"_ 
_"Default members and admins are able to remove invited members"_
https://github.com/Expensify/App?tab=readme-ov-file#workspace-chats

I believe only _Admin_ can access Members page, others access Participants page.

Reply:
> I thought this change only involved preventing the removal of default members from workspace chats.

Sure. _"Admins are not able to leave/be removed from the workspace chat"_.
https://github.com/Expensify/App?tab=readme-ov-file#workspace-chats

I noticed that automatically added users, example _guides_ accounts, are marked as _POLICY.ROLE.ADMIN_.
Therefore, I believe that this check is necessary and enough. Thoughts?

Reply:
Ah okay got it, thanks for clarifying, I'm fine with this change then 🙇